lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller.pm

```diff
@@ -17,20 +17,35 @@
 package OpenQA::WebAPI::Plugin::ObsRsync::Controller;
 use Mojo::Base 'Mojolicious::Controller';
 use Mojo::File;
-use IPC::System::Simple qw(system $EXITVAL);
+use OpenQA::WebAPI::Plugin::ObsRsync::Runner;
+
+sub init {
+    OpenQA::WebAPI::Plugin::ObsRsync::Runner::newRunner(shift);
```

Comment:
This function does not initialize the controller. Just inline that function at the single place it is used. And actually, also inline `OpenQA::WebAPI::Plugin::ObsRsync::Runner::newRunner`.

I would even inline `OpenQA::WebAPI::Plugin::ObsRsync::Runner::Run` and get rid of that entire `OpenQA::WebAPI::Plugin::ObsRsync::Runner` module.

Reply:
So you want to couple internals of Container with the Plugin and change Plugin module every time when Container needs changes? I don't think it is good idea.
The thing is that I did try different implementations of Runner, and it had internal helpers and needed quite a lot of troubleshooting to understand what was wrong and compare different implementations. Yes, currently it became quite simple but its implementation is still point of contention and I want to isolate changes inside smaller module and keep bigger module more or less stable.

Reply:
Mh... ok but then please give the functions meaningful names and follow our naming conventions. Still, in my opinion enqueuing and tracking a GRU task is something I have no problem with when it is done in the controller. If you need helper methods for doing things with Gru they should likely go into the Gru module itself like `enqueue_and_keep_track` unless it is really specific to that plugin.